packages/internal/src/build/babelPlugins/__tests__/babel-plugin-redwood-routes-auto-loader.test.ts

```diff
@@ -27,14 +20,25 @@ const transform = (filename: string) => {
   })
 }
 
-test('page auto loader correctly imports pages', () => {
-  const result = transform(getPaths().web.routes)
+describe('page auto loader correctly imports pages', () => {
+  let result: babel.BabelFileResult | null
+
+  beforeAll(() => {
+    process.env.RWJS_CWD = FIXTURE_PATH
+    result = transform(getPaths().web.routes)
```

Comment:
@jtoar This was key. We left it outside `beforeAll`, so it tried to run `getPaths()` before `RWJS_CWD` was defined